perun-ldapc/src/main/resources/perun-ldapc.xml

```diff
@@ -676,6 +676,105 @@ http://www.springframework.org/schema/aop http://www.springframework.org/schema/
 			</list>
 		</property>
 	</bean>
+	
+	<!--  
+	  This bean processes events that would lead to change of virtual attribute and audit message
+	  "virtual attribute changed". It is here to off-load processing of the change from RPC to LDAPC.  
+	-->
+	<bean id="virtualAttributeEventProcessor" class="cz.metacentrum.perun.ldapc.processor.impl.VirtualAttributeEventProcessor">
+		<property name="dispatchConditions">
+			<list>
+				<!-- group membership events -->
+				<bean class="cz.metacentrum.perun.ldapc.processor.impl.RegexpDispatchEventCondition">
+					<property name="beansCondition">
+						<list>
+							<value>cz.metacentrum.perun.core.api.Group</value>
+							<value>cz.metacentrum.perun.core.api.Member</value>
+						</list>
+					</property>
+					<property name="pattern" value=" added to Group:\[.*\]"/>
+					<property name="handlerMethodName" value="processGroupMemberChange"/>
+				</bean>
+				<bean class="cz.metacentrum.perun.ldapc.processor.impl.RegexpDispatchEventCondition">
+					<property name="beansCondition">
+						<list>
+							<value>cz.metacentrum.perun.core.api.Group</value>
+							<value>cz.metacentrum.perun.core.api.Member</value>
+						</list>
+					</property>
+					<property name="pattern" value=" validated.$"/>
+					<property name="handlerMethodName" value="processGroupMemberChange"/>
+				</bean>
+				<bean class="cz.metacentrum.perun.ldapc.processor.impl.RegexpDispatchEventCondition">
+					<property name="beansCondition">
+						<list>
+							<value>cz.metacentrum.perun.core.api.Group</value>
+							<value>cz.metacentrum.perun.core.api.Member</value>
+						</list>
+					</property>
+					<property name="pattern" value=" was removed from Group:\[.*\] totally"/>
+					<property name="handlerMethodName" value="processGroupMemberChange"/>
+				</bean>
+				<bean class="cz.metacentrum.perun.ldapc.processor.impl.RegexpDispatchEventCondition">
+					<property name="beansCondition">
+						<list>
+							<value>cz.metacentrum.perun.core.api.Group</value>
+							<value>cz.metacentrum.perun.core.api.Member</value>
+						</list>
+					</property>
+					<property name="pattern" value="expired.$|disabled.$|invalidated.$|suspended #"/>
+					<property name="handlerMethodName" value="processGroupMemberChange"/>
+				</bean>
+				<bean class="cz.metacentrum.perun.ldapc.processor.impl.RegexpDispatchEventCondition">
+					<property name="beansCondition">
+						<list>
+							<value>cz.metacentrum.perun.core.api.Member</value>
+						</list>
+					</property>
+					<property name="pattern" value=" validated.$"/>
+					<property name="handlerMethodName" value="processMemberChange"/>
+				</bean>
+				<bean class="cz.metacentrum.perun.ldapc.processor.impl.RegexpDispatchEventCondition">
+					<property name="beansCondition">
+						<list>
+							<value>cz.metacentrum.perun.core.api.Member</value>
+						</list>
+					</property>
+					<property name="pattern" value="expired.$|disabled.$|invalidated.$|suspended #"/>
```

Comment:
We recently changed member suspended message, so it should be ``suspended.$`` just like the other member states.

Reply:
Fixed.